pkg/controllers/user/monitoring/projectHandler.go

```diff
@@ -137,7 +145,7 @@ func (ph *projectHandler) deployApp(appName, appTargetNamespace string, appProje
 	appDeployProjectID := project.Name
 	clusterPrometheusSvcName, clusterPrometheusSvcNamespaces, clusterPrometheusPort := monitoring.ClusterPrometheusEndpoint()
 	clusterAlertManagerSvcName, clusterAlertManagerSvcNamespaces, clusterAlertManagerPort := monitoring.ClusterAlertManagerEndpoint()
-
+	_, clusterAlertNS := monitoring.ClusterMonitoringInfo()
```

Comment:
Is this the same as `clusterAlertManagerSvcNamespaces`, can we use a single variable

Reply:
That's why I hate to get those string from `monitoring.xxx` functions.